tests/integration/full.yaml

```diff
@@ -50,6 +50,8 @@ apps:
     enabled: false
   snapshot-controller:
     enabled: true
+  sealed-secrets:
+    enabled: false
```

Comment:
Why put it as disabled if this is full integration?

Reply:
Good catch. Thx.